src/IssueViz/Commands/IssueVizPackage.vsct

```diff
@@ -63,7 +61,7 @@
     </Buttons>
 
     <Bitmaps>
-      <Bitmap guid="guidImages" href="..\Resources\sonarlint.png"/>
+      <Bitmap guid="guidImages" href=".\Resources\sonarlint.png"/>
```

Comment:
The resources folder is now under Commands. I moved it outside of Commands when hotspots were in this project as well, so now I could return it back.